lib/reporters/verbose.js

```diff
@@ -91,6 +91,7 @@ class VerboseReporter {
 				'  ' + colors.error(runStatus.failCount, plur('test', runStatus.failCount), 'failed') :
 				'  ' + colors.pass(runStatus.passCount, plur('test', runStatus.passCount), 'passed'),
 			runStatus.knownFailureCount > 0 ? '  ' + colors.error(runStatus.knownFailureCount, plur('known failure', runStatus.knownFailureCount)) : '',
+			runStatus.pendingCount > 0 ? '  ' + colors.todo(runStatus.pendingCount, plur('test', runStatus.pendingCount), 'pending') : '',
```

Comment:
I'm sure it's on your list, but this still needs to print the titles of these tests.

Reply:
Done. I missed this originally - good catch!